### ISSUE TITLE
Feature request: securely erase/wipe a file

### ISSUE THREAD

**Issue by seadra** [arpitkh96/AmazeFileManager#1368]
An option to securely erasing/wiping files would be great.

Like a simplified version of coreutils' shred, which basically rewrites the file with random data + flush (shred defaults to x3 times) before unlinking.

I don't mean this is absolutely secure, but it's much better than a simple unlink, and the point of shred is to make the recovery harder.

**Comment by EmmanuelMess** [comment]
I don't think that this is something a file manager should do. I think this would be outside the scope of this app.

**Comment by seadra** [comment]
In a Unix CLI environment, I agree. But on Android interface, I think a file manager is a natural place for this.

**Comment by EmmanuelMess** [comment]
It would be extending the scope too much, I don't think we have the manpower to keep extending the scope. Maybe as another app/plugin that interacts with Amaze?
